.mergify.yml

```
@@ -1,8 +1,20 @@
 pull_request_rules:
+  - name: backport patches to master branch
```

I've heard this word for this kind of ports. Wdyt?
```suggestion
  - name: forward-port patches to master branch
```